### PR TITLE
Update Strings

### PR DESCRIPTION
**Pull request by deniss-mc**
**Jira issue:**
https://glia.atlassian.net/browse/MUIC-585

**Additional info:**
String updates can be seen under the task - suggestions column

### CODE REVIEW

File: widgetssdk/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
+    <string name="glia_chat_upgraded_to_video_call">Upgraded to video call</string>
     <string name="glia_chat_single_choice_card_hint">Tap on the answer above</string>
     <string name="glia_chat_not_started_hint">Send a message to start chatting</string>
-    <string name="glia_chat_file_download_success_message">File is downloaded successfully.</string>
-    <string name="glia_chat_file_download_fail_message">"File download is failed."</string>
-    <string name="glia_chat_file_open_file_title">"Open File"</string>
-    <string name="glia_chat_file_download_failed_msg">Unable to download the file at the moment, please try again later.</string>
+    <string name="glia_chat_file_download_success_message">The file has been downloaded.</string>
+    <string name="glia_chat_file_download_fail_message">Could not download the file.</string>
+    <string name="glia_chat_file_open_file_title">Open File</string>
```
Comment (jakob-mc) [inline widgetssdk/src/main/res/values/strings.xml:15]: `File` is lowercase elsewhere in the middle of the sentence

File: widgetssdk/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -56,8 +56,8 @@
     <string name="glia_dialog_engagement_ended_action_ok">@string/glia_dialog_ok</string>
 
     <!-- Overlay permissions required Dialog -->
-    <string name="glia_dialog_overlay_permissions_title">Overlay Permissions Required</string>
-    <string name="glia_dialog_overlay_permissions_message">The chat requires screen overlay permissions to show message bubbles (optional). Please enable them for your app.</string>
+    <string name="glia_dialog_overlay_permissions_title">Screen Overlay Permissions Required</string>
```
Comment (jakob-mc) [inline widgetssdk/src/main/res/values/strings.xml:59]: everything should be capitalized here? I assume this is displayed all caps in the app, but nevertheless should be correct.

**Review by jakob-mc — APPROVED** [review]

File: widgetssdk/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -70,20 +70,20 @@
...
     <!-- End screen sharing Dialog -->
     <string name="glia_dialog_screen_sharing_end_title">End Screen Sharing?</string>
-    <string name="glia_dialog_screen_sharing_end_message">Are You sure You want to end the screen sharing?</string>
+    <string name="glia_dialog_screen_sharing_end_message">Are you sure you want to end screen sharing?</string>
     <string name="glia_dialog_screen_sharing_end_cancel">@string/glia_dialog_cancel</string>
     <string name="glia_dialog_screen_sharing_end_end">End</string>
 
     <!-- Operators unavailable AlertDialog -->
-    <string name="glia_dialog_operators_unavailable_title">We\'re sorry</string>
+    <string name="glia_dialog_operators_unavailable_title">Operators Not Available</string>
```
Comment (dukhovnyi) [inline widgetssdk/src/main/res/values/strings.xml:78]: Do we need to have `Not` and `Available` uppercased?

File: widgetssdk/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -70,20 +70,20 @@
...
     <string name="glia_dialog_operators_unavailable_message">Operators are no longer available. Please try again later.</string>
 
     <!-- Unexpected error AlertDialog -->
-    <string name="glia_dialog_unexpected_error_title">We\'re sorry, there has been an unexpected error.</string>
-    <string name="glia_dialog_unexpected_error_message">Please try again later.</string>
+    <string name="glia_dialog_unexpected_error_title">Something Went Wrong</string>
```
Comment (dukhovnyi) [inline widgetssdk/src/main/res/values/strings.xml:82]: Do we need to have `Went` and `Wrong` uppercased?

File: widgetssdk/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -70,20 +70,20 @@
...
+    <string name="glia_dialog_unexpected_error_message">Please try again.</string>
 
     <!-- Permission error AlertDialog -->
-    <string name="glia_dialog_permission_error_title">We\'re sorry, one or more permission(s) are missing for required action.</string>
+    <string name="glia_dialog_permission_error_title">Additional Permissions Required</string>
```
Comment (dukhovnyi) [inline widgetssdk/src/main/res/values/strings.xml:86]: Do we need to have `Permissions` and `Required` uppercased?

Comment (deniss-mc) [reply widgetssdk/src/main/res/values/strings.xml:86]: Updated according to doc that was provided please see under task

File: widgetssdk/src/main/res/values/strings.xml (hunk 4 above)
Comment (deniss-mc) [reply widgetssdk/src/main/res/values/strings.xml:82]: Updated according to doc that was provided please see under task

File: widgetssdk/src/main/res/values/strings.xml (hunk 3 above)
Comment (deniss-mc) [reply widgetssdk/src/main/res/values/strings.xml:78]: Updated according to doc that was provided please see under task

**Pull request merged**